### DISCUSSION TITLE
有些漫画在 再漫画上能看，在zaix上不能看

### DISCUSSION THREAD

**Discussion by ALBOZ114514** [xiaoyaocz/flutter_dmzj#233] [General]
![Screenshot_2024-09-18-13-19-38-60_73ac1a6b35ecd9da6d59de244b8d1f0d](https://github.com/user-attachments/assets/b8a6693f-0a0e-4362-9b3d-15cade76aa61)

顺便问一下能不能增加漫画内单页下载图片的功能，原版dmzj有这个功能

**Comment by hanerx** [comment]
这个试了下接口里部分漫画要登录，得加token的header才能正常显示
